python/lsst/ts/nightreport/__init__.py

```diff
@@ -1,6 +1,6 @@
-# This file is part of ts_nightreport.
+# This file is part of ts-nightreport.
```

Comment:
Actually, `ts_nightreport` was correct since that is the project name. Please revert here and everywhere else where you changed that.